duckduckgo/iOS
# Move GPC to BSK

Task/Issue URL: https://app.asana.com/0/0/1203456821249397/f
Tech Design URL:
CC:

**Description**:
Move GPC code from client app to BSK

**Steps to test this PR**:
1. Open a website that supports GPC like this: http://global-privacy-control.glitch.me/
2. Check if all sections have a blue 👍
3. Disable GPC (Settings -> GPC -> Disable)
4. Go back to the site opened at step 1 and check that we have a red 👎 on header, yellow 👍 on client-side and a blue 👍 on .well-known 
5. Enable GPC back and check if all sections have a blue 👍

**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
